src/topology/algebra/floor_ring.lean

```diff
@@ -0,0 +1,225 @@
+/-
+Copyright (c) 2020 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+
+Basic topological facts (limits and continuity) about `floor`,
+`ceil` and `fract` in a `floor_ring`.
+-/
+import topology.algebra.ordered
+import algebra.floor
+
+open set function
+open_locale topological_space
+
+variables {α : Type*} [linear_ordered_ring α] [floor_ring α]
+
+lemma tendsto_floor_at_top :
+  filter.tendsto (floor : α → ℤ) filter.at_top filter.at_top :=
+begin
+  refine monotone.tendsto_at_top_at_top (λ a b hab, floor_mono hab) (λ b, _),
+  use (b:α)+((1:ℤ):α),
+  rw [floor_add_int, floor_coe],
+  exact (lt_add_one _).le
+end
+
+lemma tendsto_floor_at_bot :
```

Comment:
I'm sure all these lemmas name+statement will fit on one line after opening the `filter` namespace.